doc/libpmem2/pmem2_config_set_address.3.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEM2_CONFIG_SET_ADDRESS, 3)
+collection: libpmem2
+header: PMDK
+date: pmem2 API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2020, Intel Corporation)
+
+[comment]: <> (pmem2_config_set_address.3 -- man page for libpmem2 config API)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmem2_config_set_address**() - set address among with address request type in the
+pmem2_config structure
+
+# SYNOPSIS #
+
+```c
+#include <libpmem2.h>
+
+struct pmem2_config;
+int pmem2_config_set_address(struct pmem2_config *cfg, void *addr, int request_type);
+```
+
+# DESCRIPTION #
+
+The **pmem2_config_set_address**() function configures the address among with address request
+type which will be used for mapping. It takes one address request type only. *\*config* should
+be already initialized, please see **pmem2_config_new**(3) for details. The *\*addr* cannot be
+**NULL** and must be a multiple of the alignment required for the data source which will be used
+for mapping alongside the config. To retrieve the alignment required for specific instance of
+**pmem2_source** use **pmem2_source_alignment**(3).
+
+Possible address request types are:
+
```

Comment:
What if I want the default behavior of mmap? Shouldn't we have some type of `PMEM2_ADDRESS_HINT` or something?
Would it be ok to just always use `PMEM2_ADDRESS_FIXED_NOREPLACE`? If so, the name should probably be more general.